src/vhost.c

```diff
@@ -350,11 +364,9 @@ static int vhost_init(struct pg_brick *brick, struct pg_brick_config *config,
 	vhost_config = (struct pg_vhost_config *) config->brick_config;
 	state->output = vhost_config->output;
 	state->vid = -1;
+	state->flags = vhost_config->flags;
 	PG_PKTS_COUNT_SET(state->rx_bytes, 0);
 	PG_PKTS_COUNT_SET(state->tx_bytes, 0);
-	pg_vhost_enable(brick, enable_freacture_mask);
-	pg_vhost_disable(brick, disable_freacture_mask);
```

Comment:
These calls were failing all the time so I removed them since they don't impact when we run as server.